sentry/src/test/java/io/sentry/CombinedScopeViewTest.kt

```diff
@@ -1097,6 +1097,51 @@ class CombinedScopeViewTest {
         assertEquals(listOf("globalFingerprint"), combined.fingerprint)
     }
 
+    @Test
```

Comment:
@romtsn here's the tests for the behaviour where I'm not quite sure if this is the behaviour we want